prime-router/src/main/resources/db/migration/V24__add_processed_at.sql

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Why two files?  Combine into one.

Reply:
Flyway doesn't like it. Tried numerous ways, still no go. Talked with Ron, he agreed flyway doesn't like it.